React/src/components/Auth/index.tsx

```diff
@@ -0,0 +1,14 @@
+import LogoWolox from 'resources/img/logoWolox.png';
+
+import styles from './styles.module.scss';
+
+export function AuthContainer({ children }: any) {
```

Comment:
We can improve this type, think/check which could be the proper type for the children prop